src/cloudformation_cli_python_lib/metrics.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from typing import Any, Mapping
+
+# boto3 doesn't have stub files
+from boto3.session import Session  # type: ignore
+
+from botocore.exceptions import ClientError  # type: ignore
+
+from .interface import Action, MetricTypes, StandardUnit
+
+LOG = logging.getLogger(__name__)
+
+
+def format_dimensions(dimensions: Mapping[Any, Any]) -> Any:  # TODO fix type
+    formatted_dimensions = []
+    for key, value in dimensions.items():
+        formatted_dimensions.append({"Name": key, "Value": value})
+    return formatted_dimensions
```

Comment:
would love this to be a list comprehension

Reply:
fixed